### PR TITLE
Add Management SDK for Microsoft.DevCenter

### PR DESCRIPTION
**Pull request by tbyfield**
Adding mangement SDK Azure.ResourceManager.DevCenter for the Microsoft.DevCenter RP - this is currently in public preview.

# Contributing to the Azure SDK

Please see our [CONTRIBUTING.md](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md) if you are not familiar with contributing to this repository or have questions.

For specific information about pull request etiquette and best practices, see [this section](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#pull-request-etiquette-and-best-practices).

### CODE REVIEW

File: sdk/devcenter/ci.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# NOTE: Please refer to https://aka.ms/azsdk/engsys/ci-yaml before editing this file.
```
Comment (ArthurMa1978) [inline sdk/devcenter/ci.yml:1]: we have updated the ci.yml to ci.mgmt.yml, the also change the setting, you can refer https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/avs/ci.mgmt.yml

File: sdk/devcenter/Azure.ResourceManager.DevCenter/README.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
+# Azure DevCenter Management client library for .NET
```
Comment (ArthurMa1978) [inline sdk/devcenter/Azure.ResourceManager.DevCenter/README.md:1]: `DevCenter` to `Dev Center`

File: sdk/devcenter/Azure.ResourceManager.DevCenter/README.md [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+
+This package follows the [new Azure SDK guidelines](https://azure.github.io/azure-sdk/general_introduction.html) which provide a number of core capabilities that are shared amongst all Azure SDKs, including the intuitive Azure Identity library, an HTTP Pipeline with custom policies, error-handling, distributed tracing, and much more.
+
+## Getting started 
+
+### Install the package
+
+Install the Azure DevCenter management library for .NET with [NuGet](https://www.nuget.org/):
```
Comment (ArthurMa1978) [inline sdk/devcenter/Azure.ResourceManager.DevCenter/README.md:9]: `DevCenter` to `Dev Center`

File: sdk/devcenter/Azure.ResourceManager.DevCenter/package-lock.json [hunk 4]
```diff
@@ -0,0 +1,11 @@
+{
```
Comment (ArthurMa1978) [inline sdk/devcenter/Azure.ResourceManager.DevCenter/package-lock.json:1]: remove this file

File: sdk/devcenter/Azure.ResourceManager.DevCenter/src/Properties/AssemblyInfo.cs [hunk 5]
```diff
@@ -0,0 +1,9 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Runtime.CompilerServices;
+
+[assembly: Azure.Core.AzureResourceProviderNamespace("DevCenter")]
```
Comment (ArthurMa1978) [inline sdk/devcenter/Azure.ResourceManager.DevCenter/src/Properties/AssemblyInfo.cs:6]: Microsoft.DevCenter

File: sdk/devcenter/ci.yml (hunk 1 above)
Comment (tbyfield) [reply sdk/devcenter/ci.yml]: Renamed and updated the file

File: sdk/devcenter/Azure.ResourceManager.DevCenter/package-lock.json (hunk 4 above)
Comment (tbyfield) [reply sdk/devcenter/Azure.ResourceManager.DevCenter/package-lock.json]: Removed

File: sdk/devcenter/Azure.ResourceManager.DevCenter/src/Properties/AssemblyInfo.cs (hunk 5 above)
Comment (tbyfield) [reply sdk/devcenter/Azure.ResourceManager.DevCenter/src/Properties/AssemblyInfo.cs]: done

**Comment by tbyfield** [comment]
@ArthurMa1978 thanks for the feedback, I've updated as per your comments.

**Comment by ArthurMa1978** [comment]
Please run the `eng\scripts\Update-Snippets.ps1 devcenter` to generate the apiview file

**Comment by ArthurMa1978** [comment]
@tbyfield just realized there is a new process run, please run `eng/scripts/Update-Mgmt-CI.ps1` to update the ci setting for a new sdk

**Review by ArthurMa1978 — APPROVED** [review]

**Pull request merged**